### DISCUSSION TITLE
Dolt integration?

### DISCUSSION THREAD

**Discussion by Vectorial1024** [laravel/framework#48693] [Q&A]
I have recently encountered Dolt (https://www.dolthub.com/), which is a version controlled database that got a v1.0 a few months ago. (Ref https://www.dolthub.com/blog/2023-05-05-dolt-1-dot-0/)

Its architecture seems very revolutionary.  It would be very powerful if we can connect Dolt with the rest of the existing software workflow.

Does Dolt itself fit the philosophy of Laravel? And will Laravel be interested to investigate Dolt in the future?

**Comment by shaedrich** [comment]
Seems like you can just use it via `mysql` driver, can't you?
https://docs.dolthub.com/introduction/getting-started/database#see-the-data-in-a-sql-workbench

**Reply by juse-less** [reply]
Like @shaedrich says; it should work with the regular MySQL driver.
While I haven't tried against Laravel, database tools like TablePlus and DataGrip (either standalone or through other JetBrains IDEs) had no issues working with it as a MySQL database.
